src/index.ts

```diff
@@ -32,9 +37,26 @@ app.stage.addChild(message)
 
 const drowners = new Container()
 
-let boat, state
+let boat, state, water
 
 const setup = (loader, resources) => {
+
+  const waterFilterSprite = Sprite.from(waterFilter)
+  waterFilterSprite.texture.baseTexture.wrapMode = PIXI.WRAP_MODES.REPEAT
+  const displacementFilter = new filters.DisplacementFilter(waterFilterSprite)
+  waterFilterSprite.scale.y = 0.6
+  waterFilterSprite.scale.x = 0.6
```

Comment:
Is this going to be dependent on the size of the canvas? What happens if we make this game expand to fill the viewport of various devices?